doc/source/releases/0.8.0.rst

```diff
@@ -0,0 +1,48 @@
+=======================
+Neo 0.8.0 release notes
+=======================
+
+XXth July 2019
+
+Main added features:
+  * Proxy object implementation and new lazy mode
+  * new neo.utils module
+  
+Other features:
+  * Event and Epoch do not use array_annotations machinery
+  * deep copy annotations when slicing
+  * numpy 1.16 compatibility
+  * time_shift() method for Epoch/Event/AnalogSignal
+  * time_slice more robust
+
+New IO classes:
+  * Axograph
```

Comment:
``AxographIO`` has been around since [0.7.0](https://neo.readthedocs.io/en/latest/releases/0.7.0.html), and ``AxographRawIO`` was released recently in [0.7.2](https://neo.readthedocs.io/en/latest/releases/0.7.2.html). Both were mentioned in previous release notes, so this mention should probably be removed.